draft-ietf-scitt-architecture.md

```diff
@@ -167,29 +167,15 @@ SCITT achieves this by having producers, auditors, etc. (also called Issuers) pu
 
 {::boilerplate bcp14-tagged}
 
-# Use Cases
```

Comment:
fore reviewers, this is a move, not an edit or deletion

Reply:
its a move, under the definition of transparency